region/wreckedship/main.json

```diff
@@ -4332,12 +4541,14 @@
                 {
                   "name": "Shinespark",
                   "notable": false,
+                  "entranceCondition": {
```

Comment:
There should be `comeInShinechargeding` options too.

Reply:
Yeah true. A `leaveShineCharged` can fulfil a `comeInWithSpark`, so adding charged strats will just reduce the spark frames. I'll put it as a fixme for now